### ISSUE TITLE
Placement of opening brace in multiline object initializer calls

### ISSUE THREAD

**Issue by realbart** [dotnet/docs#35779]
For multiline object initializer calls I see multiple styles for the braces. Examples you give seem to contradict:

```
var thing = new IndexersExample {
    ...
}
```

and

```
List<Cat> cats = new List<Cat>
{
    ...
}
```

Can you either explain when to use which style (either here or on your coding style pages) or fix the examples to be consistent? Thank you!

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 0efd6a6a-b52a-f6a2-d98b-f245b55115ff
* Version Independent ID: 95025282-20a0-ded8-c6d2-3b8ffff600c4
* Content: [Object and Collection Initializers - C# Programming Guide](https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/object-and-collection-initializers)
* Content Source: [docs/csharp/programming-guide/classes-and-structs/object-and-collection-initializers.md](https://github.com/dotnet/docs/blob/main/docs/csharp/programming-guide/classes-and-structs/object-and-collection-initializers.md)
* Product: **dotnet-csharp**
* Technology: **csharp-fundamentals**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

**Comment by Adityageek** [comment]
Hi @BillWagner can I take this issue. I have experience in dotnet technologies, currently I am working as dotnet developer. So I would like to help in this issue which helps me to learn as well. Please assign this task to me. Thanks!

**Comment by ErykKorzeniowski** [comment]
Hey, I made a change @BillWagner, please review the PR

**Comment by BartoszKlonowski** [comment]
I can see that the fixing PR was not actually linked to the issue - I wonder whether it still can be made, especially by non-author comment:
Fixed by #36083

**Comment by ErykKorzeniowski** [comment]
> I can see that the fixing PR was not actually linked to the issue - I wonder whether it still can be made, especially by non-author comment:
> 
> Fixed by #36083 

It was linked. Also, PR is already merged

**Comment by BartoszKlonowski** [comment]
@ErykKorzeniowski Of course, though it depends on what you mean by linked. For me linking is when merging a PR automatically closes the linked issue, but in this case the linked (mean closing) PR was not given:
![obraz](https://github.com/dotnet/docs/assets/70535775/7ee7dc3d-b156-4cf0-89e5-dd1c87932c64)
Hence, the issue you fixed is still open.
Don't get me wrong please, my comment was not to point mistakes, but to just try if I can easily link it and push something forward, sorry for misunderstanding.

**Comment by IEvangelist** [comment]
Hi @ErykKorzeniowski and @BartoszKlonowski - the issue with the linking is that each issue number needs to be individually preceded by the fixes or closes keyword. You cannot do two issues for a single keyword. In other words, the PR was only ever going to close the first issue listed.

![image](https://github.com/dotnet/docs/assets/7679720/f271663a-2240-43c5-8942-665192b25ff5)


Closed by #36083.
